### PR TITLE
Add `rad2deg` and `deg2rad`

### LINKED ISSUE #432: Handle degree values after Coordinates refactoring
## Description

The new coordinates class will always return and expect angles in radians. We decided to add functions `deg2rad` and `rad2deg` that get N-dimensional numpy arrays and has an option to convert only specific dimensions/axis.

### PR DESCRIPTION
**Pull request by f-brinkmann**
### Which issue(s) are closed by this pull request?

Closes #432

### Changes proposed in this pull request:

- [x] add `deg2rad` and `rad2deg`
- [x] discuss import structure

will add testing if we agree on the general behavior and functionality

### CODE REVIEW

**Review by ahms5 — COMMENTED** [review]
isnt it more intuiotive to mark the axis which should be converted?

**Comment by f-brinkmann** [comment]
> isnt it more intuiotive to mark the axis which should be converted?

It's not really an axis. For example, in a 2D array it would be the columns. But I was strugelling with a good description. Was this unclear? If we want to reverse the logic, I would suggest `convert=slice(2)` as the default. But it might be harder to understand to people who are new to Python.

**Comment by f-brinkmann** [comment]
We decided to restrict the function to the pyfar case and use a parameter `domain` with possible values `'spherical'` and `'cylindircal'`

**Review by ahms5 — APPROVED** [review]
look fine, thanks for implementing. 
Just one small thing, instead of ``kind='deg2rad'`` we could directly write the multiplication factor, this would save the ``if kind=='bla'`` in ``_convert_angles()``
 but is more cosmetics

File: pyfar/classes/coordinates.py [hunk 1]
```diff
@@ -2559,6 +2559,91 @@ def cyl2cart(azimuth, height, radius):
...
+    -------
+    coordinates : numpy array
+        The converted coordinates of the same shape as the input data.
+    """
+    return _convert_angles(coordinates, domain, 180/np.pi)
+
+
+def deg2rad(coordinates, domain='spherical'):
+    """
+    Convert coordinates in degree to radians
```
Comment (ahms5) [inline pyfar/classes/coordinates.py:2594]:
```suggestion
    Convert a copy of coordinates in degree to radians
```

File: pyfar/classes/coordinates.py [hunk 2]
```diff
@@ -2559,6 +2559,91 @@ def cyl2cart(azimuth, height, radius):
     return x, y, z
 
 
+def rad2deg(coordinates, domain='spherical'):
+    """
+    Convert coordinates in radians to degree
```
Comment (ahms5) [inline pyfar/classes/coordinates.py:2564]:
```suggestion
    Convert a copy of coordinates in radians to degree
```

**Review by pingelit — APPROVED** [review]
Thanks for the implementation. I'm not quite sure if we require the `deg2rad` method. But that might just be me missing a point.

**Review by ahms5 — APPROVED** [review]

**Comment by ahms5** [comment]
my require #507

**Pull request merged**